README.md

```diff
@@ -105,22 +109,26 @@ Download/Clone the Git repository.
 
 
 ## Setup Interfaces
-Install dependencies for the interfaces, from the root of this repository, run:
```

Comment:
This feels like a step back.
I think it helps to "stay out of the specific parts and start-from-the-root" especially in this initial Getting-Started README.